std2xalpha/algorithm/comparison.d

```diff
@@ -0,0 +1,23 @@
+module std2xalpha.algorithm.comparison;
+
+// This will be an often-used idiom to fetch the prev version of a module.
+import v1 = std.algorithm.comparison;
+
+/**
+[@@@TODO@@@ Documentation specific for version 2. It should be appended to the default as
+implementation notes for this specific version.]
+*/
+alias mismatch = v1.canon!"std2xalpha".mismatch;
+
+///
+unittest
+{
+    // `ö` and `ü` are two bytes wide and both start with 0xC3
+    auto s1 = "öabc", s2 = "üabc";
+    auto a = v1.mismatch(s1, s2);
```

Comment:
Does anyone know why the CircleCI compilation errors `undefined identifier `v1`, did you mean variable `s1`?` here? It does not make any sense, `v1` is defined at line 4 as far as I can see.

Reply:
The `publictests` target verifies that documented unittests can be copy + pasted (while adding an import for the current module). The renamed import `v1` is private to the current module.

Reply:
https://github.com/andralex/phobos/pull/24

Reply:
ping @andralex

Reply:
merged, thanks